web/template/header.gohtml

```diff
@@ -1,5 +1,5 @@
 {{define "header"}}
-    <nav class="flex z-40 lg:z-50 sticky top-0 w-full items-center justify-between dark:bg-secondary bg-secondary-light p-3 h-20 z-40">
+    <nav class="flex z-40 lg:z-50 sticky top-0 w-full items-center justify-between dark:bg-secondary bg-white shadow p-3 h-20 z-40">
```

Comment:
IMO this is _very_ bright, but I don't really mind either

Reply:
I kinda like it that way. (However, I use light mode everywhere :D)

Reply:
You're the pro then 😀